README.md

```diff
@@ -6,11 +6,16 @@ DNS server uses `dnslib` and HTTP server runs on `sanic`.
 
 ## Running
 
+### From bare source
 1. Simply run `python -m pax0r_ddns_server`
 2. Set DNS NS record for domain you want to control to the the server where you are running DDNS service
 3. Post with empty body to `http://<your-server>/<domain>`. IP of the posting client will be used as dynamic IP for given domain
 4. DNS server run by this script should answer with your dynamic IP
 
+### Docker
+1. Build image : `docker build -t pax0r .`
+2. Run from image : `docker run --publish 8000:8000  pax0r `
```

Comment:
It might be worth noting that we might need to publish DNS port as well

Reply:
Done